test_runner/batch_others/test_branch_behind.py

```diff
@@ -1,6 +1,9 @@
 import subprocess
 from fixtures.zenith_fixtures import PostgresFactory, ZenithPageserver
 
+import logging
+import fixtures.log_helper  # configures loggers
+log = logging.getLogger('root')
```

Comment:
This logging setup sequence seems too complicated. Ideally, I'd like to just import `logging` or whatever and call `info`, `warning` etc. Could we do all the required setup in `log_helper`?